gp-templates/project-import.php

```diff
@@ -24,9 +24,18 @@
 		$format_options[$slug] = $format->name;
 	}
 	$format_dropdown = gp_select( 'format', $format_options, 'auto' );
+
+	$status_options = array();
+	$status_options[ 'waiting' ] = __( 'Waiting', 'glotpress' );
+	if ( $can_import ) {
+		$status_options[ 'current' ] = __( 'Current', 'glotpress' );
+	}
+	$status_dropdown = gp_select( 'status', $status_options, 'waiting' );
 ?>
```

Comment:
If there is only one option, it shouldn't be displayed as a dropdown (probably not at all).